tool/tctl/common/resource_command.go

```diff
@@ -690,28 +721,60 @@ func (rc *ResourceCommand) createToken(ctx context.Context, client auth.ClientI,
 	if err != nil {
 		return trace.Wrap(err)
 	}
-
-	err = client.UpsertToken(ctx, token)
-	return trace.Wrap(err)
+	_, err = client.GetToken(ctx, token.GetName())
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	exists := (err == nil)
+	if !rc.force && exists {
+		return trace.AlreadyExists("token '%s' already exists", token.GetSafeName())
+	}
+	if err = client.UpsertToken(ctx, token); err != nil {
+		return trace.Wrap(err)
+	}
+	fmt.Printf("token '%s' has been %s\n", token.GetSafeName(), UpsertVerb(exists, rc.force))
+	return nil
 }
 
 func (rc *ResourceCommand) createInstaller(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
 	inst, err := services.UnmarshalInstaller(raw.Raw)
 	if err != nil {
 		return trace.Wrap(err)
 	}
-
-	err = client.SetInstaller(ctx, inst)
-	return trace.Wrap(err)
+	_, err = client.GetInstaller(ctx, inst.GetName())
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	exists := (err == nil)
+	if !rc.force && exists {
+		return trace.AlreadyExists("installer '%s' already exists", inst.GetName())
+	}
+	if err = client.SetInstaller(ctx, inst); err != nil {
+		return trace.Wrap(err)
+	}
+	fmt.Printf("installer '%s' has been %s\n", inst.GetName(), UpsertVerb(exists, rc.force))
+	return nil
 }
 
 func (rc *ResourceCommand) createUIConfig(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
 	uic, err := services.UnmarshalUIConfig(raw.Raw)
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	_, err = client.GetUIConfig(ctx)
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	exists := (err == nil)
+	if !rc.force && exists {
+		return trace.AlreadyExists("UI config '%s' already exists", uic.GetName())
```

Comment:
This doesn't really make sense. There is only one UI config, and it always has the same name. I would just remove the name from the message.